openmc/geometry.py

```diff
@@ -257,7 +257,49 @@ def get_all_lattices(self):
                     lattices[cell.fill.id] = cell.fill
 
         return lattices
+    
+    def get_all_surfaces(self):
+        """
+        Return all surfaces used in the geometry
+
+        Returns
+        -------
+        collections.OrderedDict
+            Dictionary mapping lattice IDs to :class:`openmc.Surface` instances
 
+        """
+        surfaces = OrderedDict()
+        
+        for cell in self.get_all_cells().values():
+            self.get_surfaces_from_region(surfaces, cell.region)
+        return surfaces
+    
+    def get_surfaces_from_region(self, surfaces, region):
+        """
+        Recursively find all the surfaces referenced by a region and return them
+        
+        Parameters
+        ----------
+        surfaces: collections.OrderedDict
+            Dictionary mapping lattice IDs to :class:`openmc.Surface` instances
```

Comment:
I think this is a typo - doesn't the dictionary map surface IDs to `Surface` instances?